api/envoy/api/v2/route/route.proto

```diff
@@ -1076,6 +1076,20 @@ message Tracing {
   // :ref:`HTTP Connection Manager <config_http_conn_man_runtime>`.
   // Default: 100%
   envoy.type.FractionalPercent overall_sampling = 3;
+
+  message CustomTag {
+    enum Type {
+      LITERAL = 0;
+      ENVIRONMENT = 1;
+      REQUEST_HEADER = 2;
+    }
+    Type type = 1 [(validate.rules).enum.defined_only = true];
+    string tag = 2;
+    string value = 3;
+    string default_value = 4;
```

Comment:
i think you'll want some brief documentation around these field values - its not clear while reading the API why there would be a "value" and a "default_value" defined next to each other

Reply:
added